README.md

```diff
@@ -60,6 +60,7 @@ If you would like to be notified of future conferences that would be added to th
 | [LaravelLive India](https://laravellive.in/) | 19 Mar | New Delhi | A PHP Laravel Conference. |
 | [PyDelhi Conf](https://conference.pydelhi.org/) | 18-19 Mar | IIM Lucknow Noida Campus, Noida | PyDelhi conference is hosted annually by PyDelhi Community with an aim to promote Python programming language. |
 | [FOSSMeet](http://fossmeet.in/2017/) | 10-12 Mar | NIT Calicut | FOSSMeet is an annual event on Free and Open source software, conducted at National Institute of Technology, Calicut. |
+|[KDE Conference](https://conf.kde.in/)| 10-12 Mar | IIT Guwahati | It is a platform for Qt/KDE contributors, developers and enthusiasts to come together, share their knowledge. | 
```

Comment:
Can you update the venue as `IIT Guwahati, Assam`?

Reply:
`enthusiasts to come together, share their knowledge.` -> `enthusiasts to come together and share their knowledge.`

Reply:
yeah sure :)